docs/content/en/docs/fleet-manager/rollout/canary.md

```diff
@@ -0,0 +1,410 @@
+  creationTimestamp: "2024-01-11T02:39:03Z"
+  finalizers:
+  - apps.kurator.dev
+  generation: 1
+  name: rollout-demo
+  namespace: default
+  resourceVersion: "6853"
+  uid: 42ac8bce-7f6d-4bd5-90d9-c85aa696fbe5
+spec:
+  source:
+    gitRepository:
+      gitImplementation: go-git
+      interval: 3m0s
+      ref:
+        branch: master
+      timeout: 1m0s
+      url: https://github.com/stefanprodan/podinfo
+  syncPolicies:
+  - destination:
+      fleet: quickstart
+    kustomization:
+      force: false
+      interval: 0s
+      path: ./deploy/webapp
+      prune: true
+      timeout: 2m0s
+    rollout:
+      port: 9898
+      rolloutPolicy:
+        rolloutTimeoutSeconds: 600
+        trafficAnalysis:
+          checkFailedTimes: 2
+          checkIntervalSeconds: 90
+          metrics:
+          - intervalSeconds: 90
+            name: request-success-rate
+            thresholdRange:
+              min: 99
+          - intervalSeconds: 90
+            name: request-duration
+            thresholdRange:
+              max: 500
+          webhooks:
+            command:
+            - hey -z 1m -q 10 -c 2 http://backend-canary.webapp:9898/
+            timeoutSeconds: 60
+        trafficRouting:
+          canaryStrategy:
+            maxWeight: 50
+            stepWeight: 10
+          gateways:
+          - istio-system/public-gateway
+          hosts:
+          - backend.webapp
+          timeoutSeconds: 60
+      serviceName: backend
+      testLoader: true
+      trafficRoutingProvider: istio
+      workload:
+        apiVersion: apps/v1
+        kind: Deployment
+        name: backend
+        namespace: webapp
+  - destination:
+      fleet: quickstart
+    kustomization:
+      force: false
+      interval: 5m0s
```

Comment:
?I remeber we cannot set it

Reply:
This is another syncPolicy that can set the kustomization.interval. It will not affect the sync of other kustomization resources.